File: legal-informatic-backend/src/main/java/org/example/legalinformaticbackend/dto/RulingTemplateDrDeviceLegalCaseDTO.java
package org.example.legalinformaticbackend.dto;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;

@Data
@Getter
@Setter
public class RulingTemplateDrDeviceLegalCaseDTO {
    private DrDeviceLegalCaseDTO drDeviceLegalCaseDTO;
    // Response koji sam dobio od endpoint-a za rasudjivanje po pravilima
    private String ruling;
}

File: legal-informatic-frontend/src/app/model/RulingTemplateDrDeviceLegalCaseDTO.ts
import {DrDeviceLegalCaseDTO} from "./DrDeviceLegalCaseDTO";

export interface RulingTemplateDrDeviceLegalCaseDTO{
  drDeviceLegalCaseDTO: DrDeviceLegalCaseDTO;
  ruling: string;
  date?: string;
  judgementMeta?: string;
  explanationMeta?: string;
}
